### DISCUSSION TITLE
Question on active status

### DISCUSSION THREAD

**Discussion by JulianHSC** [community/community#127076] [Repositories]
### Select Topic Area

General

### Body

How long can I leave my github account inactive without the github teams removing the account? That means I do not login for 3-6 months and it still exist. Is there a time limit? Will I get any warning or questions in my email requesting me to login to keep it active?

**Comment by davevad93** [comment]
Hi @JulianHSC ,

As far as I know GitHub has only a policy for inactive `Enterprise` accounts as stated in the [docs](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/managing-dormant-users) that says the following: 
- `A user account is considered to be dormant if it has not been active for 90 days.`

 Regarding personal accounts GitHub has a [username policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy) that states the following: 
- `GitHub account names are available on a first-come, first-served basis, and are intended for immediate and active use.`

It's not very clear what “active use” means, as there's no specification on how long it takes for a user to be considered “inactive”. I think if you don't log in for 3-6 months your account will be fine, I have a friend who hasn't logged in for almost 2 years because he lost his password and is no longer interested in using GitHub and despite that his account is still up.
